platform-tests/example-apps/healthcheck/Godeps/Godeps.json

```diff
@@ -1,10 +1,7 @@
 {
-	"ImportPath": "healthcheck",
+	"ImportPath": "github.com/alphagov/paas-cf/platform-tests/example-apps/healthcheck",
```

Comment:
You're theory seems to be right, when I moved paas-cf under my GOPATH it changed it for me too. What is a bigger mystery is why the `Packages` property on lines 5-7 was removed. Was that done manually? I don't think these changes will break anything, so happy to keep them in, but they may get changed back in future depending on who's doing the vendoring.